profile/README.md

```diff
@@ -67,7 +68,7 @@ documentation, developer references, and quickstart guides will always be locate
 
 # Community
 
-We invite you to contribute! Check out the issues in the individual repositories for
+We invite you to contribute! Check out the Issues in the individual repositories for
```

Comment:
This is probably better, but Dave's concern was around the reference at line 86.